tests/test_backbones/test_tnt.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+import torch
+from torch.nn.modules.batchnorm import _BatchNorm
+
+from mmcls.models.backbones import TNT
+
+
+def check_norm_state(modules, train_state):
+    """Check if norm layer is in correct train state."""
+    for mod in modules:
+        if isinstance(mod, _BatchNorm):
+            if mod.training != train_state:
+                return False
+    return True
+
+
+def test_tnt_backbone():
```

Comment:
Place all backbone unit tests in `tests/test_models/test_backbones`. Refers to #321